app/uk/gov/hmrc/statepension/services/StatePensionService.scala

```diff
@@ -197,6 +198,14 @@ trait NpsConnection extends StatePensionService {
       true
     else false
   }
+
+  final val CHANGE_SPA_MIN_DATE = new LocalDate(1970, 4, 6)
+  final val CHANGE_SPA_MAX_DATE = new LocalDate(1978, 4, 5)
+
+  private def checkStatePensionAgeUnderConsideration(dateOfBirth: LocalDate): Boolean = {
+    (dateOfBirth.equals(CHANGE_SPA_MIN_DATE) || dateOfBirth.isAfter(CHANGE_SPA_MIN_DATE)) && (dateOfBirth.equals(CHANGE_SPA_MAX_DATE) || dateOfBirth.isBefore(CHANGE_SPA_MAX_DATE))
```

Comment:
Is it good to use this condition:
!dateOfBirth.isBefore(CHANGE_SPA_MIN_DATE) && !dateOfBirth.isAfter(CHANGE_SPA_MAX_DATE)